var/spack/repos/builtin/packages/ghostscript/package.py

```diff
@@ -71,3 +74,9 @@ def build(self, spec, prefix):
     def install(self, spec, prefix):
         make('install')
         make('soinstall')
+
+    @classmethod
+    def determine_version(cls, exe):
+        output = Executable(exe)('--version', output=str, error=str)
```

Comment:
I think this is still missing:
```suggestion
        output = Executable(exe)('--help', output=str, error=str)
```
This is what I see on my dev box:
```console
$ gs --version
9.26
$ gs --help
GPL Ghostscript 9.26 (2018-11-20)
Copyright (C) 2018 Artifex Software, Inc.  All rights reserved.
Usage: gs [switches] [file1.ps file2.ps ...]
[ ... ]
```